setup.cfg

```diff
@@ -42,9 +42,11 @@ exclude =
 
 [options.extras_require]
 dev =
+    coverage
     elasticsearch-dsl~=7.4
-    pytest~=7.0
-    pytest-cov~=4.0
+    importlib-resources
```

Comment:
iirc, this is part of python 3.7+ and a backport before that.
Maybe add "; python<=x.y" as needed?

https://docs.python.org/3.7/library/importlib.html#module-importlib.resources

Reply:
It is a backport but now for more than just 3.7. It now has the updated [interface](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files) introduced in 3.9; the old [interface](https://docs.python.org/3/library/importlib.resources.html#deprecated-functions) is now marked as depreciated and hence I like to just stick to the new one. 

So I can limit it to 3.9 but it is minorly annoying because we will now have to check the python version for every import vs the backport is which is perfectly functional for all currently supported python versions.

```python
if sys.version_info >= (3, 9):
    from importlib import resources
else:
    import importlib_resources as resources
```

Reply:
@wrkhenddher any thoughts on how you'd like me to move forward? I don't want to merge this with this pending as it affects the review that follows too.

Reply:
Nice! all good!